src/python/system/path_patcher.py

```diff
@@ -34,10 +37,18 @@ def _patch_single(obj, attr, fn):
   setattr(obj, attr, fn)
   setattr(getattr(obj, attr), '__path_patcher__', True)
 
+  if obj == builtins and six.PY2:
```

Comment:
this was needed because setattr(__builtins__, ...) no longer works after these fixes. On Python 3, setting builtins.<attr> does the same thing, so isn't needed there.